packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -42,9 +42,15 @@ export const styles = theme => ({
     overflowX: 'hidden',
     width: '100%',
   },
-  /* Styles applied to the tablist element if `variant="scrollable"`. */
+  /* Styles applied to the tablist element and its hidden scrollbar vendor prefixed if `variant="scrollable"`. */
   scrollable: {
     overflowX: 'scroll',
+    msOverflowStyle: 'none',
+    overflow: '-moz-scrollbars-none',
+    scrollbarWidth: 'none',
```

Comment:
I believe we only need to hide it for MacOS. What do you think of this diff: https://github.com/mui-org/material-ui/issues/14706#issuecomment-486146295?

Reply:
I tested on macOS chrome and Firefox. Working great. This even works for Windows IE. Do we use any other technique on other OS? `'&::-webkit-scrollbar': {
      display: 'none'
     },` for chrome and safari macOS

`scrollbarWidth: 'none'` for firefox macOS
`overflow: '-moz-scrollbars-none'` i think for firefox other versions macOS
`msOverflowStyle: 'none'` for IE windows- if we dont use any other technique

Reply:
Notice the existing mechanism in place to hide the scrollbar.

Reply:
Again existing mechanism of negative margin handles outer scrollbar. Thus, works fine for windows. Do you want me to write OS-specific style here?

Reply:
There is other way around also. We can keep overflow-x: hidden and still be able to scroll using buttons.

Reply:
@Umerbhat 
Would that work with touch devices though?

You want touch devices to be able to scroll the tablist by swiping

Reply:
Yes overflow:hidden should be done with condition like (Desktop only) && scrollbar height ==0, if we follow that approach else, we have easier CSS approach. we can even write this CSS for inner scrollbar only.